src/Command/MigrateCommand.php

```diff
@@ -255,14 +398,46 @@ private function createExclusionChecker($path, array $excludePaths)
          * @param string $path
          * @return bool
          */
-        return static function ($path) use ($excludePaths) {
+        $exclusions[] = static function ($path) use ($excludePaths) {
             foreach ($excludePaths as $excludePath) {
                 if (strpos($path, $excludePath) !== false) {
                     return true;
                 }
             }
             return false;
         };
+
+        return $exclusions;
+    }
+
+    /**
+     * @param string[] $excludePaths
+     * @param string $path
+     * @param callable[] $exclusions
+     * @return array The $exclusions array with the new exclusion filter
```

Comment:
`callable[]` instead of `array`

Reply:
No longer relevant with the most recent refactoring.